src/main/python/util.py

```diff
@@ -3,6 +3,7 @@
 import socket
 import docker
 import datetime
+import requests
```

Comment:
Nit: Instead of importing the whole `requests` library it might be better to just import the `ConnectionError` exception like so:
```python
from reqests.exceptions import ConnectionError as RequestsConnectionError
```

Reply:
smart - nice catch.  updated